src/pytorch_lightning/plugins/precision/apex_amp.py

```diff
@@ -55,35 +54,26 @@ def dispatch(self, trainer: "pl.Trainer") -> None:
             self._connected = True
         return super().dispatch(trainer)
 
-    def backward(
-        self,
-        model: "pl.LightningModule",
-        closure_loss: Tensor,
-        optimizer: Optional[Optimizer],
-        optimizer_idx: Optional[int],
-        *args: Any,
-        **kwargs: Any,
-    ) -> None:
+    def backward(self, tensor: Tensor, model: "pl.LightningModule", *args: Any, **kwargs: Any) -> None:
```

Comment:
`closure_` does not apply to Lite necessarily, but why not keep it as `loss` instead of just `tensor`?

Reply:
In Lite, you can call backward on anything that is a tensor. It doesn't have to be a scalar value like in PL that represents a loss (it could be a score that needs to be maximized). As long as it has a graph attached. In PL, one would have to override LM.backward() for this I suppose.